index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./css/style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.9.0/css/all.min.css" integrity="sha512-q3eWabyZPc1XTCmF+8/LuE1ozpg5xxn7iO89yfSOd5/oKvyqLngoNGsx8jq92Y8eXJ/IRxQbEC+FGSYxtk2oiw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <title>Raymond's Portfolio</title>
+</head>
+<body>
+  <header>
+    <!-- Navigation Bar -->
+    <nav>
+      <a href="#" class="logo">
+        Ray
+        <img src="./images/r_logo.png" alt="logo" width="20px" height="20px">
+      </a>
```

Comment:
- [ ] Great job making the logo a link 👏 Kindly move it outside the `nav` tag for better semantics since the `nav` tag is for navigation and that doesn't include the logo 👍

Reply:
Thanks for the review,  should I wrap both the nav element and the logo element in a section tag, because I tried doing it as you mentioned, and it made a mess of my style 😅, unless I take the headline section off the header.
And another question please, after resolving all these issues, all I need to do is to just make a new commit , right?  @ha-manel

Reply:
@Raykingflamez Yes, you can take the headline off the header, some people say the header is only for the navigation bar and some say it includes the headline too, so I'm saying they're both correct ways of writing code 🤔 

After you fix all of the issues, just push your commits and submit the PR for review in your dashboard 😊 

Best of luck ⭐

Reply:
@ha-manel  Thanks 😊, just one more question, please pardon me, should I make another PR using this branch or another and should the commits be this changes here or my initial commits after I'm done making the changes?
Sorry I'm being such a novice, but I appreciate your kindness

Reply:
@Raykingflamez Never be sorry for asking questions, we're here to help! 😄 

so after you work on the changes locally (on this branch) just write commits and push them to this branch, you'll see your commits appear in this pull request. That's it! no need for another branch or another pull request 😁 

let me know if anything is still unclear 😊

Reply:
It's all clear now,  thanks @ha-manel  for explaining 😊
I appreciate it.
I'll go ahead and make all the changes now

Reply:
Awesome! good luck 😊